src/test/java/com/app/AppTest.java

```diff
@@ -0,0 +1,42 @@
+package com.app;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.NoSuchElementException;
+
+public class AppTest {
+  @Test
+  public void testRun() {
+    // Avoid the menu to be shown on the screen
+    ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(outContent));
+
+    // Input 1 in the menu
+    String input = "1";
```

Comment:
1 may not be ideal to test with when adding more options

Reply:
Fixed